chart/templates/carto-google-api-service-account.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: carto
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "carto.commonGcpApi.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- if eq .Values.workloadIdentityConfig.enableWorkloadIdentity "true" }}
+    iam.gke.io/gcp-service-account: {{ .Values.workloadIdentityConfig.workloadIdentitySaEmail | quote }}
+    {{- end }}
+automountServiceAccountToken: {{ .Values.workspaceApi.serviceAccount.automountServiceAccountToken }}
```

Comment:
That line is using something from workspaceApi. If that is required, we should define it at a global level.